examples/translate-all/README.md

```diff
@@ -0,0 +1,39 @@
+# “Translate All Content” Sidebar Widget
+
+This widget translates text of all the entry's fields from the default locale to all
+other locales enabled in the space, using the
+[Yandex translation API](https://translate.yandex.com/developers).
+
+## Bootstrap example for local development
+
+In the directory containing this README, run
+```bash
+npm install
+```
+
+First set the access token on your environment, then create the widget on Contentful:
+```bash
+export CONTENTFUL_MANAGEMENT_ACCESS_TOKEN=<contentfulManagementApiToken>
+contentful-widget create --space-id <yourSpaceId>
+```
+
+Serve on `http://localhost:3000`:
+```bash
+gulp watch
```

Comment:
You need to built the widget api first, that is run `make` in the repo’s main dir.